tests/unit/test_syscontainers.py

```diff
@@ -35,6 +35,38 @@ def wrapper(*args, **kwargs):
 
     patch = fake_patch
 
+@unittest.skipIf(no_mock, "Mock not found")
+class TestSystemContainers_do_checkout(unittest.TestCase):
+    """
+    Unit tests for refactored function from SystemContainers.do_checkout method.
+    """
+    def test_get_remote_location(self):
+        """
+        We create temp directories in tmp folder, then remove it after the test completes
+
+        """
+        # Create directories for 3 different cases '_get_remote_location' is checking
+        try:
+            os.makedirs('/tmp/test-remote/rootfs/usr')
```

Comment:
instead of hardcoding `/tmp`, please use `tempfile.mkdtemp`.  Also all the code using it must be in a try/finally block, where the finally does the `shutil.rmtree` of the temporary directory created

Reply:
ah, sounds good, thanks for the suggestion, will start wrapping entire code section under a try/finally block.